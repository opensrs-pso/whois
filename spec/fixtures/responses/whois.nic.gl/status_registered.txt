
Domain Information
Query: google.gl
Status: Delegated
Created: 11 Mar 2003
Modified: 05 Dec 2012
Expires: 01 Jan 2014
Name Servers:
    ns1.google.com
    ns2.google.com

Registrar Information
Registrar Name: MarkMonitor
Registration URL: http://www.markmonitor.com
Address: 391 N Ancestor Place
Boise, ID 83704 US
Country: US
Customer Service Contact: Matt Serlin
Customer Service Email: ccops AT markmonitor DOT com




Registrant:
Name: Google Inc.
Address:
	1600 Amphitheatre Parkway
	Mountain View, CA 94043
	US
Email Address: dns-admin AT google DOT com
Phone Number: +1 6303300100

