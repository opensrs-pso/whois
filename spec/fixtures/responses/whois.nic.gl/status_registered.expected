#domain
  should: %s == "google.gl"

#domain_id
  should: %s raise_error(Whois::AttributeNotSupported)


#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2003-03-11")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-12-05")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2014-01-01")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "MarkMonitor"
  should: %s.organization == nil
  should: %s.url          == "http://www.markmonitor.com"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(2)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.google.com"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.google.com"
